### ISSUE TITLE
UI / PubSub | Display premilinary image and some data

### ISSUE THREAD

**Issue by theusaf** [Soundbendor/smart-bin-app#8]
## User Story
As a Smart Compost Bin User, I want to see pictures or information about scans taken by my bin so that I know that I am contributing to the project, and that the process is working.

### Why?
This feature will please our user by showing a quick image and some initial information to let the user know that things are working in the background, which will help keep the user engaged.

### Who?
This feature is for the user.

### Can we support it?
The application should be able to support it, although it depends on external infrastructure to be developed. A testable version with mock data could be done.

## Solution/Requirements
* When a new image is taken by a bin, an entry should appear in the app 
* It should appear in a list or something else and should be obvious that the overall task is not complete
* The image should appear relatively quickly 

## Dependencies
* Partially blocked by bin connectivity
* Blocked by missing external infrastructure

## Estimate 
* Initial design - 1 week
* Proof of concept - 1 week
* Complete implementation - TBD (see dependencies)

## When this Issue is Done
This Issue is complete once:
* UI elements are displayed on a triggering event (picture and data captured by bin)
* It works with real data
* Required verifications from team members are made and relevant pull requests/commits are merged into `dev`/`main`
